contents/docs/web-analytics/revenue-tracking.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Revenue tracking
+sidebar: Docs
+showTitle: true
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+> **Important:** This feature is currently in beta. Please leave a comment on this page if you have feedback or questions, or leave feedback via the in-app support modal.
+
+Our revenue tracking feature enables you to capture revenue from your website via the events you are already sending us, or an integration with a payment platform, which you can then track in your [web analytics dashboard](https://app.posthog.com/web).
+
+## Choosing a revenue data source
+
+PostHog can collect revenue from two sources:
+
+### Option 1: Properties on events
+
+The simplest way to send us revenue data is to add it as properties on revenue generating events, such as when a purchase is completed, a user starts a subscription, or a subscription is renewed.
+
+When you do so, you need to include two properties:
+
+- `value`: The amount of revenue generated by the event.
+- `currency`: The currency of the revenue in ISO 4217 format (USD, EUR, GBP, etc.).
+
+You can choose any name you like for these properties – you can customize them in the **Revenue** tab in [data management](https://app.posthog.com/data-management/revenue).
+
+You `currency` property is technically optional as you can choose a static currency when configuring your revenue event in your dashboard, but we recommend sending it if you can to ensure backward compatibility should you ever take payments in a different currency.
```

Comment:
The `currency` property is technically optional as you can choose a static currency when configuring your revenue event in your dashboard, but we recommend sending it if you can to ensure backward compatibility should you ever take payments in a different currency.
```